src/BcPrebidVast.js

```diff
@@ -0,0 +1,365 @@
+/**
+ * Header Bidding Plugin Brightcove module.
+ */
+
+// CONSTANTS
+
+var _vjs = window.videojs !== undefined ? window.videojs : null;
+var _prebidGlobal = require('./PrebidGlobal.js');
+var _vastManager = require('./VastManager.js');
+var _prebidCommunicator = require('./PrebidCommunicator.js');
+var _logger = require('./Logging.js');
+var _prefix = 'PrebidVast->';
+
+var $$PREBID_GLOBAL$$ = _prebidGlobal.getGlobal();
+
+_logger.always(_prefix, 'Version 0.0.10');
+
+var BC_prebid_in_progress = $$PREBID_GLOBAL$$.plugin_prebid_options && $$PREBID_GLOBAL$$.plugin_prebid_options.biddersSpec;
+
+// the function does bidding and returns bids thru callback
+function doPrebid(options, callback) {
+	if ($$PREBID_GLOBAL$$.bc_pbjs && options.biddersSpec) {
+		  var pbjs = $$PREBID_GLOBAL$$.bc_pbjs || {};
+	      pbjs.que = pbjs.que || [];
```

Comment:
it's confusing to use `$$PREBID_GLOBAL$$` and `pbjs`. Can you explain how this works?

Reply:
I will modify this code to avoid confusion.